test/jmap/email/sort_list_email_test.dart

```diff
@@ -276,6 +276,16 @@ void main() {
 
   group('sort list email test', () {
 
+    test('sort list email by receivedAt descending', () async {
+      final listEmailResponse = await getListEmailAndSortBy(EmailComparator(EmailComparatorProperty.receivedAt)..setIsAscending(false));
```

Comment:
why not create separate method `setAscending` and `setDescending`

Reply:
IMO, We need the `bool` value here. This shouldn't be too complicated. Let's think more simply.